tools/api_key_tool.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+import glob
+import re
+import argparse
+
+tokenString = '[Token '
+fixturesDirectory = 'tests/fixtures/'
+zeroapiregex = r'(\[Token )0{40}(\])'
+anyapiregex = r'(\[Token ).{40}(\])'
+zeroapistring = '[Token '+40*'0'+']'
+
+
+def api_key_detector(file):
+    '''
+    Detect whether the file contains an api key in the Token object that is not 40*'0'.
+    See issue #86.
+    :param file: path-to-file to check
+    :return: boolean
+    '''
+    f = open(file, 'r')
+    text = f.read()
+    if re.search(anyapiregex, text) is not None and  \
+            re.search(zeroapiregex, text) is None:
+        return True
+    return False
+
+
+def api_key_remover(file):
+    '''
+    Change the api key in the Token object to 40*'0'.  See issue #86.
+    :param file: path-to-file to change
+    '''
+    f = open(file, 'r')
```

Comment:
This be written more compactly as 

```python
with open(filename, 'r') as fp:
    text = fp.read()
```

This implicitly closes the file handler once you leave the `with` block. However, this works too :).